app/models/user.rb

```diff
@@ -14,6 +14,15 @@ class User < ApplicationRecord
   validates :birthday, :gender, presence: true
   validate :birthday_within_last_100_years
 
+  def self.digest(string)
+    cost = if ActiveModel::SecurePassword.min_cost
+             BCrypt::Engine::MIN_COST
+           else
+             BCrypt::Engine.cost
+           end
+    BCrypt::Password.create string, cost: cost
```

Comment:
Just need one if key in the hash same variable 
```suggestion
    BCrypt::Password.create(string, {cost})
```